packages/terra-data-grid/src/DataGrid.jsx

```diff
@@ -598,11 +616,11 @@ const DataGrid = forwardRef((props, ref) => {
           columnHeaderHeight={columnHeaderHeight}
           rowHeight={rowHeight}
           rowHeaderIndex={rowHeaderIndex}
+          onCellSelect={onCellSelect}
```

Comment:
Did we need to move this?  It was previously with the other selection callbacks.

Reply:
This was an abandoned attempt at alphabetization, reverted.